install/create_certs.sh

```diff
@@ -0,0 +1,22 @@
+
+
+web_ingress_public_ip=$(jq -r '.nodes[] | select(has("web_ingress")) | .web_ingress.public_ip' config.json)
+web_ingress_ssh_username=$(jq -r '.nodes[] | select(has("web_ingress")) | .web_ingress.ssh_username' config.json)
+
+web_ingress_ssh_command="ssh -o StrictHostKeyChecking=no $web_ingress_ssh_username@$web_ingress_public_ip"
+
+certs_dir="/opt/neuraldb_enterprise/certs"
+
+$shared_file_system_ssh_command <<EOF
+    sudo apt install openssl
+    sudo mkdir -p $certs_dir
+    cd $certs_dir
+    openssl req -x509 -newkey rsa:4096 -keyout traefik.key -out traefik.crt -days 365 -nodes -subj "/CN=$web_ingress_public_ip"
+
+    cat <<EOT | sudo tee $certs_dir/certificates.toml
+[[tls.certificates]]
```

Comment:
This is a good idea, I'll try to to do this. The only hiccup is that I'll need to check if the traefik job can access the certs with these permissions, not sure how docker would interact with these permissions.

Reply:
Just checked and traefik.key is already permission 600 on creation through openssl, and that's the other files (e.g. traefik.cert) is 622, which is fine because this is a public certificate anyways. So I think it's fine the way it is right now.